PLAYER/SetPlayerHealthRechargeLimit.md

```diff
@@ -9,8 +9,10 @@ aliases: ["0xC388A0F065F5BC34"]
 void _SET_PLAYER_HEALTH_RECHARGE_LIMIT(Player player, float limit);
 ```
 
-
 ## Parameters
-* **player**: 
-* **limit**: 
+- **player**: PlayerId()
+- **limit**: float number between 0.1 and 0.9
+
+## Important
```

Comment:
Documentation does not support additional headings. Please include this as description.